client/src/utils/Constants/Money.ts

```diff
@@ -0,0 +1,83 @@
+import { AmountsKeys, ForeignAmountsKeys, ZlotyAmountsKeys } from '@/utils';
+
+// Money mass
+export const JEDEN_GR = 1.64;
```

Comment:
```suggestion
export const JEDEN_GR_MASS_IN_GRAMS = 1.64;
```
Lepiej w nazwie consta niż w komentarzu którego IDE nie trawi. 

Może też być cos w stylu GRAMS_PER_JEDEN_GR

Analogicznie pozostałe.

Reply:
A no i pytanie czy nie lepiej po angielsku.

Reply:
Wprowadzę proprawkę